### ISSUE TITLE
for some reason gl file rabel-dietitian is empty..

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/groceristar-fetch#186]

**Comment by sanchit94** [comment]
I think it has been empty all this while!

**Comment by atherdon** [comment]
you maybe right. i just can recollect it in my memory

**Comment by atherdon** [comment]
will try to search up to old commits about this file.
Are you back to us, Sanchit?

**Comment by sanchit94** [comment]
Yeah I tried searching it too, cannot find it in history of the master branch though!

**Comment by sanchit94** [comment]
Am I back? Even I don't know that! :)

**Comment by atherdon** [comment]
haha :0 will be happy to have you with us again.

I actually find the source of data, and commit is related to Jule 12th.
scrolling into it.
And happy to know that my approach with keeping everything at git can help
to crack it...

**Comment by atherdon** [comment]
We all are beginners ;) this project is plain JS(let's assume it nodejs) but we also setting up some middleprojects that actually has an express core. i think we can start from here, and then slowly move to express stuff. what do you think?

**Comment by atherdon** [comment]
this actually the link to main source of that grocery list: https://rebelrd.com/shopping-lists/
i'll dig more and i assume i'll find at least partially converted content into suitable format

**Comment by atherdon** [comment]
Jul, 12th is our day for cathing it: https://github.com/GroceriStar/groceristar-fetch/commits/master?after=c57b18aa4e08af0f8221153a2d6d640413e1a6d6+384

**Comment by sanchit94** [comment]
Looks good Arthur! Hope we can get something out of this. Why don't you assign me small tasks that are not on priority, so that I can get hands on with this stuff!

**Comment by atherdon** [comment]
noted @sanchit94 i have so low-level, but really hard core tasks
